spec/validators/claim/base_claim_sub_model_validator_spec.rb

```diff
@@ -27,7 +27,7 @@
       @misc_fee = FactoryBot.create :misc_fee,:with_date_attended, claim: claim
       FactoryBot.create :date_attended, attended_item: @misc_fee
       claim.fees.map(&:dates_attended).flatten      # iterate through the fees and dates attended so that the examples below know they have been created
-      claim.form_step = :fees
+      claim.form_step = :fixed_fees
```

Comment:
I think this spec is going to need work to make sure validation of each fee association is covered - currently this looks like it is testing fixed fee validation yet it has basic fees (fees of these two fee types should not co-exist).

Reply:
actually, i recommend above to change this step name which would resolve this